internal/core/queries/get_dd_by_id_with_rels.go

```diff
@@ -62,6 +64,19 @@ func (ch GetDeviceDefinitionWithRelsQueryHandler) Handle(ctx context.Context, qu
 				Region:       di.Region,
 				Capabilities: common.JSONOrDefault(di.Features),
 			})
+
+			if !di.R.Integration.ManufacturerTokenID.IsZero() {
+				if m, ok := di.R.Integration.ManufacturerTokenID.Int64(); ok {
+					manufTknID := uint64(m)
+					integrations[len(integrations)-1].ManufacturerTokenID = &manufTknID
```

Comment:
This read strangely to me. I think instead of

```
for _, a := range v {
	w = append(w, f(a))
	w[len(w)-1].x = g(a)
}
```

it might read better as

```
for _, a := range v {
	b := f(a)
	b.x = g(a)
	w = append(w, b)
}
```